openhtf/io/frontend/__main__.py

```diff
@@ -96,47 +96,41 @@ class StationStore(threading.Thread):
   """
   def __init__(self):
     super(StationStore, self).__init__()
-    self._death_event = threading.Event()
+    self._stop_event = threading.Event()
```

Comment:
I hear what you're saying, but the method that sets this event is literally called `Stop()`...